packages/keymaps/src/browser/keybindings-widget.tsx

```diff
@@ -94,6 +97,9 @@ export class KeybindingWidget extends ReactWidget {
      * The regular expression used to extract values between fuzzy results.
      */
     protected readonly regexp = /<match>(.*?)<\/match>/g;
+    /**
+     * The keybinding separator.
```

Comment:
the "regex" of keybinding separator ?

Reply:
Thanks, I've updated the documentation to better reflect the code.